src/Tuura/Function.hs

```diff
@@ -0,0 +1,47 @@
+{-# LANGUAGE TypeFamilies #-}
+module Tuura.Function (
+    printBooleanFunction
+    ) where
+
+import Tuura.Graph
+
+-- data Function = Open String | Closed String deriving (Eq, Ord)
+
+instance Graph Function where
```

Comment:
I just realised that we should probably switch to using `alga` library for working with graphs. But that's future work.

Reply:
I agree actually, it would make some stuff really nice and neat :)